runners/spark/src/main/java/org/apache/beam/runners/spark/util/BroadcastHelper.java

```diff
@@ -31,84 +30,42 @@
 /**
  * Broadcast helper.
  */
-public abstract class BroadcastHelper<T> implements Serializable {
+public class BroadcastHelper<T> implements Serializable {
```

Comment:
This class seems to suffer some alignment issues, probably from removing the abstract and inner implementations.